### DISCUSSION TITLE
I can't see the focus on the Editor.

### DISCUSSION THREAD

**Discussion by cupid20103** [ueberdosis/tiptap#5425] [General Questions]
Hello, @everyone!

I am using Tiptap Editor to build the Chat system.
But I can't see the cursor when I focus the Editor. (The content is empty.)
If the content is full, I can see the cursor.

Why did this happen? And how to solve this problem?

**Comment by bdbch** [comment]
With cursor you mean the `|` bracket?

Do you have a short code snippet how you have setup the editor?

**Comment by abecirovic3** [comment]
It might be that some widths of your elements are 0 when there is no content. I had an issue like that and solved it I believe by setting the min-width of some element to 1px.
